en/app.ftl

```diff
@@ -465,11 +473,17 @@ waitlist-control-email-placeholder = yourname@example.com
 waitlist-control-country-label = What country do you live in?
 waitlist-control-country-label-2 = What country or region do you live in?
 waitlist-control-locale-label = Select your preferred language.
+# Deprecated
 waitlist-submit-label = Join the Waitlist
+waitlist-submit-label-2 = Join the waitlist
+# Deprecated
+waitlist-privacy-policy-agree = By clicking “{ waitlist-submit-label }”, you agree to our <a>Privacy Policy</a>.
 # Variables:
 #  $joinButtonLabel (string): the value of waitlist-submit-label
-waitlist-privacy-policy-agree = By clicking “{ waitlist-submit-label }”, you agree to our <a>Privacy Policy</a>.
+waitlist-privacy-policy-agree-2 = By clicking “{ waitlist-submit-label-2 }”, you agree to our <a>Privacy Policy</a>.
```

Comment:
I wasn't sure if this warranted a new string, since the only change is the variable - another approach would be to update the variable reference in all localised files.

Reply:
We do need it as the string, if localized, won't change to version 2 in the variable.

My question is that, don't we put `$joinButtonLabel` in the variable instead?

Reply:
Oh oops, that comment is incorrect. At first I used a regular variable, but then I realised that I could just reference the other string directly, i.e. I want to include this localised value into the string: https://github.com/mozilla-l10n/fx-private-relay-l10n/blob/905234e205030376d650965902d949e8ef9b168d/en/app.ftl#L477-L478

Removed in 5d5ce8f.

Reply:
> We do need it as the string, if localized, won't change to version 2 in the variable.
> 
> My question is that, don't we put `$joinButtonLabel` in the variable instead?

Yes, the variable usually has the `$` in it.

Reply:
That took care of the "problem", :). Thanks for looking into all the issues.